widget/embedded/src/utils/wallets.ts

```diff
@@ -492,23 +509,6 @@ export function tokensAreEqual(
   );
 }
 
-export function getDefaultToken(
```

Comment:
@Ikari-Shinji-re I removed this functionality. I'm not sure it's needed anymore. if it is, we should revert back this change.